build/stf-run-ci/tasks/deploy_stf.yml

```diff
@@ -40,6 +40,9 @@
                 persistent:
                   storageClass: {{ __service_telemetry_storage_persistent_storage_class }}
       {% endif %}
+              certificates:
+                endpoint_cert_duration: {{ __service_telemetry_events_certificates_endpoint_cert_duration }}
+                ca_cert_duration: {{ __service_telemetry_events_certificates_ca_cert_duration }}
```

Comment:
Ah I think I figured out the problem you mentioned earlier.

snake_case vs camelCase here (should use camelCase for the ServiceTelemetry manifest)